git-send-email.perl

```diff
@@ -920,7 +935,7 @@ sub validate_address {
 			valid_re => qr/^(?:quit|q|drop|d|edit|e)/i,
 			default => 'q');
 		if (/^d/i) {
-			return undef;
+			return;
```

Comment:
On Fri, 10 Jun 2016, Philip Prindeville wrote:

> > @@ -920,7 +935,7 @@ sub validate_address {
> >  			valid_re => qr/^(?:quit|q|drop|d|edit|e)/i,
> >  			default => 'q');
> >  		if (/^d/i) {
> > -			return undef;
> > +			return;
> 
> Unless you're dropping the address, in which case there wouldn't be one.

It is funny. Elsewhere this PR introduce parentheses and return statements
where they could be considered unnecessary and distracting. And here, an
explicit return value (the `validate_address` function *is* expected to
return a value) is removed, puzzling every reader.

If I was asked to accept this patch, I would not.